src/pages/settings/index.tsx

```diff
@@ -1,6 +1,26 @@
 import Head from "next/head"
+import { styled } from "linaria/react"
 
 import TopBar from "../../components/TopBar"
+import SettingItem from "../../components/SettingItem"
+
+const Main = styled.main`
+  display: flex;
+  flex-direction: column;
+  gap: var(--space-s);
```

Comment:
flex gap isn't fully supported yet, it's really annoying: https://caniuse.com/?search=flex%20gap

In this case I've been using a bit of a hack as gap is widely supported for grids:

```css
display: grid;
grid-auto-flow: row; /* This sets the children to implicit rows in the grid */
gap: 8px;
```